osu.Game/Online/Chat/MessageNotifier.cs

```diff
@@ -129,7 +129,16 @@ private void checkForMentions(Channel channel, Message message)
         /// Checks if <paramref name="message"/> contains <paramref name="username"/>.
         /// This will match against the case where underscores are used instead of spaces (which is how osu-stable handles usernames with spaces).
         /// </summary>
-        private static bool checkContainsUsername(string message, string username) => message.Contains(username, StringComparison.OrdinalIgnoreCase) || message.Contains(username.Replace(' ', '_'), StringComparison.OrdinalIgnoreCase);
+        // Converted the message string to an array of strings split by (' ') and checked if the message contains the username of the user , an attempt to fix  issue #15693
```

Comment:
We do not use comments to reference bugs. The correct way to leave a trace that this change was made to fix a bug is to add a test case for it. See `TestSceneMessageNotifier`.